### ISSUE TITLE
``Provided object is not a type literal`` with `clientExtensions` preview feature enabled

### ISSUE THREAD

**Issue by NADESHIKON** [arthurfiorette/prisma-json-types-generator#12]
```
generator client {
  provider        = "prisma-client-js"
  previewFeatures = ["orderByNulls", "clientExtensions"]
}

datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
}
```

```
> prisma generate

Environment variables loaded from .env
Prisma schema loaded from prisma\schema.prisma
Error: 
✔ Generated Prisma Client (4.9.0 | library) to .\..\..\node_modules\.pnpm\@prisma+client@4.9.0_prisma@4.9.0\node_modules\@prisma\client in 116ms

Provided object is not a type literal: AnimePayload['scalars']
```

**Comment by arthurfiorette** [comment]
Can you share a reproducible example?

**Comment by NADESHIKON** [comment]
> Can you share a reproducible example?

Sorry, the codebase I currently have is closed source so I can't really share one. But I believe it happens when you enable the preview feature `clientExtensions` since it works if I take it off.

**Comment by arthurfiorette** [comment]
You can create a sample code in a open source repository, just so I can reproduce myself your problem.

**Comment by NADESHIKON** [comment]
https://github.com/NADESHIKON/prisma-json-bug-reproduce
![image](https://user-images.githubusercontent.com/20222292/215354899-647ea1c9-e458-42ba-9ed1-c8ada126466d.png)

**Comment by arthurfiorette** [comment]
Thanks for you repo.

The problem was because clientExtensions generates completely different model schemas.
Just released `2.2.0` with the fix, can you try it out?

**Comment by arthurfiorette** [comment]
Hey @NADESHIKON, thanks for your time.

The problem was because `clientExtensions` generates completely different model schemas.
Just released `2.2.0` with the fix, can you try it out?

**Comment by NADESHIKON** [comment]
> Hey @NADESHIKON, thanks for your time.
> 
> The problem was because `clientExtensions` generates completely different model schemas. Just released `2.2.0` with the fix, can you try it out?

Thank you.
However, I am getting unsupported object field error:
```You had an object fields, but they aren't supported yet.
Can you open a issue at https://github.com/arthurfiorette/prisma-json-types-generator?
```

**Comment by arthurfiorette** [comment]
Yes, I put them because `<model>Payload` types generates `scalars` and `object` fields, I couldn't reproduce an example that had objects filled, that's why I created this warning.

It just warns that the `object` property wasn't modified.

**Comment by arthurfiorette** [comment]
I can solve this if you modify your example repo to be more similar to your original code that has this objects property.

**Comment by NADESHIKON** [comment]
> I can solve this if you modify your example repo to be more similar to your original code that has this objects property.

https://github.com/NADESHIKON/prisma-json-bug-reproduce/commit/bc2508e1f1af12f27fd75344d2d96e78da1a89f6
Here you go

**Comment by arthurfiorette** [comment]
Hey @NADESHIKON, just released 2.2.1 with the warning removed, it just did not need to do anything, so I removed it :)

Sorry for the delayed response, you can reopen this issue if needed.
